### DISCUSSION TITLE
About creating a UI for training of custom NER Data

### DISCUSSION THREAD

**Discussion by imhans33** [explosion/spaCy#9845] [Help: Coding & Implementations] [answered]
I am working with custom NER data which is now trained from CLI. I want to create a small ui for starting the training and viewing the training progress. Is it possible to build ui for my needs or only can be done form CLI. I didnt find much help in docs mentioning about this. Thanks in advance

**Comment by adrianeboyd** [comment, marked as answer]
Sure, the CLI commands are basically just wrappers around training functions that you could also call directly from your UI. You could also write a custom training logger to provide the output you want in your UI during training, too. There are some example loggers in the `spacy-loggers` package: https://github.com/explosion/spacy-loggers
